style.css

```diff
@@ -919,7 +919,7 @@ section {
 .section-inner {
 	margin-left: auto;
 	margin-right: auto;
-	max-width: 120rem;
```

Comment:
This helper class is used for other elements than the search modal, so you can change this line back to `120rem`.

Reply:
Sorry, its back to 120rem now.